pkgs/development/libraries/catboost/default.nix

```diff
@@ -79,24 +97,45 @@ stdenv.mkDerivation (finalAttrs: {
   cmakeFlags = [
     "-DCMAKE_BINARY_DIR=$out"
     "-DCMAKE_POSITION_INDEPENDENT_CODE=on"
-    "-DCATBOOST_COMPONENTS=app;libs${lib.optionalString pythonSupport ";python-package"}"
+    "-DCATBOOST_COMPONENTS=app;libs${lib.optionalString pythonSupport ";python-package"}${lib.optionalString rLibrary ";R-package"}"
   ] ++ lib.optionals cudaSupport [
-    "-DHAVE_CUDA=on"
+    "-DHAVE_CUDA=yes"
```

Comment:
I don't mind changing it, but I want to keep it consistent.

Reply:
[Catboost's latest documentation](https://catboost.ai/en/docs/installation/build-native-artifacts) indicates yes|no. Is there any reason to go with on|off instead?

Reply:
There is no particular reason for on/off, just personal preference. (I really wanted to use uppercase, but for some reason, it's lowercase.)